<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectionEditor</class>
 <widget class="QWidget" name="SelectionEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>474</width>
    <height>260</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>390</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="SourceLayout">
     <property name="spacing">
      <number>5</number>
     </property>
     <item>
      <widget class="QLabel" name="SourceLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>The dataset for which selections are saved</string>
       </property>
       <property name="text">
        <string>Data Producer</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="SourceInfo">
       <property name="toolTip">
        <string>The dataset for which selections are saved</string>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {background-color: snow; border: 1px inset grey;}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="ElementTypeLayout">
     <property name="spacing">
      <number>9</number>
     </property>
     <item>
      <widget class="QLabel" name="ElementTypeLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>The element type of the saved selections</string>
       </property>
       <property name="text">
        <string>Element Type</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="ElementTypeInfo">
       <property name="toolTip">
        <string>The element type of the saved selections</string>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {background-color: snow; border: 1px inset grey;}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="ExpressionLayout">
     <property name="spacing">
      <number>29</number>
     </property>
     <item>
      <widget class="QLabel" name="ExpressionLabel">
       <property name="toolTip">
        <string>Specify the expression which defines the relation between the saved selections using boolean operators: !(NOT), &amp;(AND), |(OR), ^(XOR) and ().</string>
       </property>
       <property name="text">
        <string>Expression</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="Expression">
       <property name="toolTip">
        <string>Specify the expression which defines the relation between the saved selections using boolean operators: !(NOT), &amp;(AND), |(OR), ^(XOR) and ().</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>456</width>
        <height>122</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="pqExpandableTableView" name="PropertiesView"/>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayoutToolbar">
         <item>
          <widget class="QToolButton" name="AddActiveSelection">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="toolTip">
            <string>Add active selection</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../../Widgets/Resources/QtWidgets.qrc">
             <normaloff>:/QtWidgets/Icons/pqPlus.svg</normaloff>:/QtWidgets/Icons/pqPlus.svg</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="RemoveSelectedSelection">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="toolTip">
            <string>Remove selected selection from the saved selections. Remember to edit the Expression.</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../../Widgets/Resources/QtWidgets.qrc">
             <normaloff>:/QtWidgets/Icons/pqMinus.svg</normaloff>:/QtWidgets/Icons/pqMinus.svg</iconset>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="VerticalSpacerToolbar">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="RemoveAllSelections">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="toolTip">
            <string>Remove all saved selections</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../../Widgets/Resources/QtWidgets.qrc">
             <normaloff>:/QtWidgets/Icons/pqDelete.svg</normaloff>:/QtWidgets/Icons/pqDelete.svg</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="FindResetCleanLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QPushButton" name="ActivateCombinedSelections">
       <property name="focusPolicy">
        <enum>Qt::TabFocus</enum>
       </property>
       <property name="toolTip">
        <string>Set the combined saved selections as the active selection</string>
       </property>
       <property name="text">
        <string>Activate Combined Selections</string>
       </property>
       <property name="icon">
        <iconset resource="../../../Components/Resources/pqComponents.qrc">
         <normaloff>:/pqWidgets/Icons/pqApply.svg</normaloff>:/pqWidgets/Icons/pqApply.svg</iconset>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>pqExpandableTableView</class>
   <extends>QTableView</extends>
   <header>pqExpandableTableView.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../Widgets/Resources/QtWidgets.qrc"/>
  <include location="../../../Components/Resources/pqComponents.qrc"/>
 </resources>
 <connections/>
</ui>
